controller/rest/workload.go

```diff
@@ -262,7 +262,21 @@ func handlerWorkloadListBase(apiVer string, w http.ResponseWriter, r *http.Reque
 		return
 	}
 
-	cached := cacher.GetAllWorkloads(view, acc)
+	// get POST body
+	idlist := utils.NewSet()
+	if r.Method == http.MethodPost {
```

Comment:
Could query.brief be true when POST("/v2/workload") is requested?
Then the payload is ignored & returns in line 232?

Reply:
discussed with the use case with UI team. The query.brief will not be used in this case.